src/khoj/interface/web/chat.html

```diff
@@ -1868,6 +2054,62 @@
                         <div id="connection-status-icon" style="width: 10px; height: 10px; border-radius: 50%; margin-right: 5px;"></div>
                         <div id="connection-status-text" style="margin: 5px;"></div>
                     </div>
+                    <div style="border-top: 1px solid black; ">
```

Comment:
If the user doesn't have any documents indexed, we can create a little section here with a link to https://docs.khoj.dev/category/clients/